src/org/usfirst/frc/team2335/robot/commands/AutoDriveTo.java

```diff
@@ -0,0 +1,43 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class AutoDriveTo extends Command
+{
+    public AutoDriveTo()
+    {
+        requires(Robot.driveTrain);
+        requires(Robot.ultraSound);
+    }
+
+    protected void initialize()
+    {
+    	
+    }
+
+    protected void execute()
+    {
+    	
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
+    protected boolean isFinished()
+    {
+        return Robot.ultraSound.atZeroInches();
```

Comment:
Will it really be at zero inches, since the ultrasound sensor is within the robot frame?